src/Backend/Modules/Pages/Js/Pages.js

```diff
@@ -744,23 +744,18 @@ jsBackend.pages.extras = {
    * Creates the html for a text field
    */
   getTextFieldHtml: function (text, label, key) {
-    var html = '<div class="panel panel-default" id="user-template-text-' + key + '">'
-
-    html += '<div class="panel-heading">'
-    html += '<h2 class="panel-title">' + label + '</h2>'
-    html += '</div>'
-
-    html += '<div class="panel-body">'
-
-    html += '<div class="form-group last">'
-    html += '<input data-ft-label="' + label + '" type="text" class="form-control" value="' + text + '" />'
-    html += '</div>'
-
-    html += '</div>'
-
-    html += '</div>'
-
-    return html
+    return `
```

Comment:
is this supported in all browsers?

Reply:
It's this right? https://caniuse.com/#feat=template-literals

Reply:
Yeah indeed

> ✅ Use a ES6 Template Literals to return the snippet of HTML. We lose IE11 support by doing that, but I guess we should stop supporting IE11 in the backend? ⚠️ This is the backend and any admin is most likely using an evergreen browser? Should I make all the methods that return html in this class, use template literals?


I wonder if we should allow more modern Javascript, knowing that this is the backend of the CMS, and should we still support IE11?

Reply:
Yes, we should support IE11.
I understand that you want to use modern JS, but this is too small of an advantage to justify dropping IE11 support.

Reply:
I share Mathias opinion, i think we should not drop the support for IE11

Reply:
This PR is a good solution for an annoying problem but dropping the support for IE11 seems to me a little bit too early.

Reply:
Ok, sounds good. Yeah, this MR is a pretty small change, and I wasn't aware that IE11 support is still needed in the backend of the CMS, so I wanted to discuss it here. Some CMS still support IE11 (enterprise-focused ones and wordpress), some more modern CMS systems drop their support. Like [Craft CMS requirements](https://docs.craftcms.com/v3/requirements.html#control-panel-browser-requirements) or [Shopify requirements](https://help.shopify.com/en/manual/intro-to-shopify/shopify-admin/supported-browsers). But I understand that we don't want to bundle/transpile the core js code + each module's javascript file either, to make it easier for devs, so we have to keep our javascript like it's 2008 😁. 

Anyway, I'll make some changes to just use trim without ES2015 features :-)